spec/arrays.dd

```diff
@@ -744,6 +752,10 @@ array.length = i;
         input from the console - it's unlikely to be longer than 80.
         )
 
+$(H3 $(LNAME2 capacity-reserve, `capacity` and `reserve`))
+
+        $(P The $(D capacity) property determines how many elements can be appended
```

Comment:
This doesn't tell what `capacity` actually is: the `length` the array can grow to before needing to reallocate.

Reply:
Maybe mention that the specific amount of elements is `capacity - length`

Reply:
Now done.